server/pom.xml

```diff
@@ -11,6 +11,67 @@
   <artifactId>keywhiz-server</artifactId>
   <name>Keywhiz Server</name>
 
+  <profiles>
+    <profile>
+      <id>container</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>sql-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>create-keywhiz-database</id>
+                <phase>none</phase>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>
+        no-custom-mysql-port
```

Comment:
is the `no-custom-mysql-port` just copied from a different place or entirely new?

Reply:
That's copied from the model sub-project. I figured the flexibility made sense to preserve even though I don't believe we're taking advantage of it.

Reply:
could be just some old cruft